runtime/wasm/runtime.js

```diff
@@ -360,23 +360,24 @@
     var buffer = caml_buffer?.buffer
     var out_buffer = buffer&&new Uint8Array(buffer,0,buffer.length)
 
-    start_fiber = wrap_fun(
-        {parameters: ['eqref'], results: ['externref']},
-        caml_start_fiber, {promising: 'first'}
-    )
-    var _initialize = wrap_fun(
-        {parameters: [], results: ['externref']},
-        _initialize, {promising: 'first'}
-    )
-    var process = globalThis.process;
-    if(process && process.on) {
-        process.on('uncaughtException', (err, origin) =>
-            caml_handle_uncaught_exception(err))
-    }
-    else if(globalThis.addEventListener){
-        globalThis.addEventListener('error', event=>
-            event.error&&caml_handle_uncaught_exception(event.error))
-    }
-    await _initialize();
-})(((joo_global_object,jsoo_exports,globalThis)=>(x)=>eval("("+x+")"))(globalThis,globalThis?.module?.exports||globalThis,globalThis),
-   PRIMITIVES, STRINGS);
+  start_fiber = wrap_fun(
+    {parameters: ['eqref'], results: ['externref']},
+    caml_start_fiber, {promising: 'first'}
+  )
+  var _initialize = wrap_fun(
+    {parameters: [], results: ['externref']},
+    _initialize, {promising: 'first'}
+  )
+  var process = globalThis.process;
+  if(process && process.on) {
+    process.on('uncaughtException', (err, origin) =>
+      caml_handle_uncaught_exception(err))
+  }
+  else if(globalThis.addEventListener){
+    globalThis.addEventListener('error', event=>
+      event.error&&caml_handle_uncaught_exception(event.error))
+  }
+  await _initialize();
+})(((joo_global_object,jsoo_exports,globalThis)=>(x)=>{return eval("("+x+")")})(globalThis,globalThis?.module?.exports||globalThis,globalThis),
+   PRIMITIVES, STRINGS,
+   ((joo_global_object,jsoo_exports,globalThis)=>FRAGMENTS)(globalThis,globalThis?.module?.exports||globalThis,globalThis))
```

Comment:
This, also...

Reply:
Not sure what happened here. I think it should be right now?

Reply:
It’s not.

Reply:
@vouillon I don’t really understand this file, so I don’t remember what was the right way to do this.